index.html

```diff
@@ -1,10 +1,50 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <link rel="stylesheet" href="style.css">
     <title>Simple Donut</title>
   </head>
   <body>
-    <h1>Coming Soon</h1>
+    <header>
+      <h1>What is Simple Donut?</h1>
+      <p>Simple Donut is a percentage chart that can be toggled from pie to donut.
```

Comment:
This way it matches the next two lines
```suggestion
      <p>
        Simple Donut is a percentage chart that can be toggled from pie to donut.
```